### PR TITLE
Revision update

### PR DESCRIPTION
**Pull request by johli**
### Description of your changes

- Updated README with detailed installation and usage instructions.
- Added new sh script to automatically install environment variables.
- Added new sh script to download borzoi model weights and annotation files.
- Added borzoi tutorials (data processing, training, variant scoring, gradients).
- Added new jupyter notebook examples with fancy coverage track & annotation plots.
- Added option to run multi-fold script with fold 3 only (multiple crosses).
- Added environment variable with conda sh path.
- Added additional environment variables for auxiliary data (such as annotations).
- Re-factored and cleaned up gradient scripts.
- Added installation with .toml.
- Added missing multi-fold variant scoring scripts.
- Added missing data processing scripts.

### Type of change
- [X] Bug fix
- [X] New feature
- [X] Refactoring / code clean-up
- [X] Documentation add / update

### (If applicable) How has this been tested?

Tests:
- Re-ran all scripts from 'borzoi-paper' repo and 'borzoi' tutorials.
- Checked that all major results from the paper reproduced (test evals, e/s/paQTLs).

Environment:
- Python 3.10
- TensorFlow 2.15

### CODE REVIEW

**Review by davek44 — APPROVED** [review]
I'm just going to trust you on this one :)
Let me know if there's anything in particular you want me to look at.
